newIDE/app/src/Profile/AdditionalUserInfoDialog.js

```diff
@@ -0,0 +1,287 @@
+              floatingLabelText={
+                <Trans>What would you use GDevelop for?</Trans>
+              }
+              translatableHintText={t`Select one`}
+              value={gdevelopUsage}
+              onChange={(e, i, newUsage: string) => {
+                setGdevelopUsage(newUsage);
+              }}
+              fullWidth
+              disabled={editInProgress}
+            >
+              {gdevelopUsageOptions.map(usageOption => (
+                <SelectOption
+                  key={usageOption.value}
+                  value={usageOption.value}
+                  primaryText={usageOption.label}
+                />
+              ))}
+            </SelectField>
+            <SelectField
+              floatingLabelText={<Trans>What kind of work do you do?</Trans>}
+              translatableHintText={t`Select one`}
+              value={currentWork}
+              onChange={(e, i, newWork: string) => {
+                setCurrentWork(newWork);
+              }}
+              fullWidth
+              disableUnderline
+              disabled={editInProgress}
+            >
+              {currentWorkOptions.map(workOption => (
+                <SelectOption
+                  key={workOption.value}
+                  value={workOption.value}
+                  primaryText={workOption.label}
+                />
+              ))}
+            </SelectField>
+            <SelectField
+              floatingLabelText={
+                <Trans>Have you used other no-code engines?</Trans>
+              }
+              translatableHintText={t`Select one`}
+              value={noCodeExperience}
+              onChange={(e, i, newEnginesUsage: string) => {
+                setNoCodeExperience(newEnginesUsage);
+              }}
+              fullWidth
+              disableUnderline
+              disabled={editInProgress}
+            >
+              {creationExperienceOptions.map(experienceOption => (
+                <SelectOption
+                  key={experienceOption.value}
+                  value={experienceOption.value}
+                  primaryText={experienceOption.label}
+                />
+              ))}
+            </SelectField>
+            <SelectField
+              floatingLabelText={
+                <Trans>How did you hear about GDevelop?</Trans>
+              }
+              translatableHintText={t`Select one`}
+              value={hearFrom}
+              onChange={(e, i, newHearFrom: string) => {
+                setHearFrom(newHearFrom);
+              }}
+              fullWidth
+              disableUnderline
+              disabled={editInProgress}
+            >
+              {hearFromOptions.map(hearFromOption => (
+                <SelectOption
+                  key={hearFromOption.value}
+                  value={hearFromOption.value}
+                  primaryText={hearFromOption.label}
+                />
+              ))}
+            </SelectField>
```

Comment:
Should those 4 select fields be within a map? To avoid the setting of the not-changing props in 4 different places

Reply:
Or those not-changing props could be an object that is spread in each SelectField

Reply:
yeah that would improve the readability 👍

Reply:
actually, because each selectfield depends on a state value, I'm not sure it makes it simpler.
I'll leave it like this for the moment